### PR TITLE
Cybnon/12/extend trust anchor firmware capabilitys

### PR DESCRIPTION
**Pull request by Cybnon**

### CODE REVIEW

File: src/OCC/valise/valise.cpp [hunk 1]
```diff
@@ -255,4 +263,54 @@ void routeValiseCborEcho(OSCMessage &msg, int addressOffset)
...
+/**
+ * Retrieve the address family from the device's memory
+ *
+ * @return The stored address family. Sending over OSC as string
+ */
+void routeValiseAddrFamilyGet(OSCMessage &msg, int addressOffset)
+{
+    Preferences valise;
+    valise.begin("vault", false);
+    String addrFamily = valise.getString("addr_family", "bc"); // Default to "bc" if not set
```
Comment (eckelj) [inline src/OCC/valise/valise.cpp:310]: Should there be a 'bc' parameter?
or jut a comment?

File: src/OCC/utils/utils.cpp [hunk 2]
```diff
@@ -144,4 +144,17 @@ std::vector<uint32_t> getPath(char *pathStr)
         token = strtok(NULL, "/");
     }
     return myPath;
+}
+
+/**
+ * Sends an error message using OSC protocol.
+ *
+ * @param address The address to send the error message to.
```
Comment (eckelj) [inline src/OCC/utils/utils.cpp:152]: There is no address in the method. old method docs? or something missing? where is the message sent to?

**Review by eckelj — APPROVED** [review]

File: src/OCC/valise/valise.cpp (hunk 1 above)
Comment (Cybnon) [reply src/OCC/valise/valise.cpp:310]: this is for the differentiation of mainnet and tested for libwally core functions

File: src/OCC/utils/utils.cpp (hunk 2 above)
Comment (Cybnon) [reply src/OCC/utils/utils.cpp]: thanks i missed that one. this part of the doc is removed now

**Pull request merged**